server/webapp/WEB-INF/rails.new/webpack/views/dashboard/dashboard_widget.js.msx

```diff
@@ -81,7 +81,7 @@ const DashboardWidget = {
         <div class="dashboard" key="dashboard-content">
           <PersonalizedViewsWidget vm={personalizeVM} />
           {messageView}
-          {_.map(vnode.attrs.dashboard.getPipelineGroups(), (pipelineGroup) => {
+          {_.map(vnode.attrs.dashboard.getPipelineGroupsForView(personalizeVM), (pipelineGroup) => {
```

Comment:
It doesn’t feel quite right to pass the entire VM here. I would figure we pass an actual DashboardFilter object instead. That would break any weird cross dependencies or circular ones. That said I haven’t sat down to verify the resultant dependency graph, so I’m speaking purely from instinct here. Even if there are no dependency issues, it still feels more correct to pass in a filter rather than an object that will delegate a level or two down. If you feel strongly otherwise, we can talk about it.